docs/src/main/tut/kleisli.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: default
+title:  "Kleisli"
+section: "data"
+source: "https://github.com/non/cats/blob/master/core/src/main/scala/cats/data/Kleisli.scala"
+scaladoc: "#cats.data.Kleisli"
+---
+# Kleisli
+Kleisli is a data type that will come in handy often, especially if you are working with monadic functions.
```

Comment:
Most importantly... just some thoughts that you can discard as you wish; this _is_ a good tutorial.

The idea, that I concede I may have failed to correctly portray, is that everything in the introduction _is_ covered in later sections (or rather, _must_ be covered), but in a simpler tone; nothing more, nothing less